cfn-resources/stream-instance/cmd/resource/mappings.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resource
+
+import "go.mongodb.org/atlas-sdk/v20231115007/admin"
+
+func NewStreamsTenant(model *Model) *admin.StreamsTenant {
+	dataProcessRegion := *model.DataProcessRegion
+	streamConfig := model.StreamConfig
+	streamTenant := &admin.StreamsTenant{
+		Name:    model.InstanceName,
+		GroupId: model.ProjectId,
+		DataProcessRegion: &admin.StreamsDataProcessRegion{
+			CloudProvider: *dataProcessRegion.CloudProvider,
+			Region:        *dataProcessRegion.Region,
+		},
+	}
+	if streamConfig != nil {
```

Comment:
```suggestion
	if streamConfig := model.StreamConfig: streamConfig != nil {
```
normally better to restrict variable scope to where it's needed

Reply:
thanks, done!